src/sortStudents.ts

```diff
@@ -1,16 +1,50 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades'
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function getAverageGrade(person: Student): number {
+  return person.grades.reduce((sum, mark) => (
+    sum + mark), 0) / person.grades.length;
+}
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  return [...students].sort((a, b) => {
+    switch (sortBy) {
+      case ('name'):
```

Comment:
for cases, you can use your enum. Otherwise, enum is useless, you don't use it.
```suggestion
      case SortType.Name:
```

Reply:
Thnx! Changed. Enum on maximum now 😎